src/components/tickets/ticket.ts

```diff
@@ -1,9 +1,17 @@
 import { Model } from 'objection'
 
+export enum statusType { 
```

Comment:
In JS it's a convention to use PascalCase for enums

Reply:
Thanks for the advice. I changed it.